### DISCUSSION TITLE
How can I throttle or slow down the pace how renovate creates PR's?

### DISCUSSION THREAD

**Discussion by rvillane** [renovatebot/renovate#32260] [Request Help] [answered]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitHub Enterprise Cloud (github.com)

### Please tell us more about your question or problem

Hello, I have q renovate bot that scans a large collection of GHE repos, and renovate does a fantastic job creating the PR's with new dependencies detected, but each PR triggers a CI/CD pipeline execution causing resource contention as the PR creation pace goes faster than the CI/CD system autoscaling.

Is there a way to throttle or pause the renovate bot execution between GHE repos? for example, when the bot completes PR's for one GHE repo, wait X minutes before starting to process the next GHE repo?  Certainly I'm open to other suggestions as well.  I did look discussion https://github.com/renovatebot/renovate/discussions/14500 but is a different situation as in that case, the problem is multiple PR's created for the same GHE repo, but I don't same that problem, a typical execution is one PR per GHE repo.

thanks

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
I don't think there's any solution for that today. Please "Suggest an idea" with proposed syntax and logic

**Reply by rvillane** [reply, marked as answer]
thanks
